Restaurant/ViewModels/ChefLookingAtOrderViewModel.cs

```diff
@@ -26,7 +26,7 @@ public class ChefLookingAtOrderViewModel : BaseViewModel
         public ICommand CompleteDishCommand { get; }
 
 		public ChefLookingAtOrderViewModel(int orderNumber,
-               IEnumerable<OrderItemViewModel> orderItems,
+               Order order,
```

Comment:
or remove "int orderNumber"
or change "Order order" to "IEnumerable<Dish> orderItems" (order.Dishes)